src/includes/blogpost.njk

```diff
@@ -5,13 +5,12 @@
     </div>
   </div>
   <aside class="blogpost-sidebar">
-    <dl class="blogpost-meta">
-      <dt class="updated">Updated</dt>
-      <dd class="updated">{{ postDate }}</dd>
+    <div class="blogpost-meta">
+      <p class="updated">Updated {{ postDate }}</p>
       {% if postAuthor %}
-      <dt class="author">By</dt>
-      <dd class="author">{{ postAuthor }}</dd>
+      <p class="author">By {{ postAuthor }}</p>
```

Comment:
Any reason to switch away from the DL? A DL seems like a better fit for the content?